compute_endpoint/globus_compute_endpoint/cli.py

```diff
@@ -534,6 +539,53 @@ def delete_endpoint(*, name: str, force: bool, yes: bool):
     Endpoint.delete_endpoint(ep_dir, get_config(ep_dir), force=force)
 
 
+@app.command("self-diagnostic")
+@click.option(
+    "-z",
+    "--gzip",
+    "compress",
+    default=False,
+    is_flag=True,
+    help="Save the output to a Gzip-compressed file.",
+)
+@click.option(
+    "--log-size",
+    default=5,
+    help=(
+        "Specify the number of megabytes (MB) to read from log files."
+        " Defaults to 5MB."
+    ),
+)
```

Comment:
It might be useful to specify kilobyte sizes here as well. Maybe a separate click option?

Reply:
For simplicity, I think we should have just one log size option. I'll refactor to use KB instead.